src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +47,11 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> p.getAge() >= fromAge
+                        && p.getAge() <= toAge
+                        && p.getSex().equals(Person.Sex.MAN))
```

Comment:
Please replace all your s, n, p etc with proper full name. After 2 weeks you will not remember what that abbreviation means
```suggestion
                .filter(person -> person.getAge() >= fromAge
                        && person.getAge() <= toAge
                        && person.getSex().equals(Person.Sex.MAN))
```